lib/DllPlugin.js

```diff
@@ -20,6 +20,10 @@ class DllPlugin {
 	constructor(options) {
 		validateOptions(schema, options, "Dll Plugin");
 		this.options = options;
+
+		if (!options.format) {
```

Comment:
Don't modify the passed object. This isn't needed anyway, just remove it.

Reply:
I have added this check to always have default value for format flag.

If user doesn't send any format option, by default it will go into else case and execute the current functionality. I have removed the code as you have suggested.